frontend/apps/staff/src/lib/reservation-units/[pk]/form.ts

```diff
@@ -68,14 +75,17 @@ const PricingFormSchema = z.object({
   isActivatedOnBegins: z.boolean(),
   materialPriceDescriptionFi: z
     .string()
+    .transform((x) => formatWhitespace(x))
```

Comment:
Is the whitespace also cleaned on the data sent to backend or is this just for validation?